ownCloud/Resources/de-DE.lproj/Localizable.strings

```diff
@@ -39,6 +39,15 @@
 "Authenticated as %@ via %@" = "Authentifiziert als 1%@ über 1%@";
 "Edit" = "Bearbeiten";
 "Credentials" = "Zugangsdaten";
+"Rejected" = "Abgelehnt";
```

Comment:
@hosy I am a bit surprised to see german translation here, since I was thinking those are maintained separately in transifex

Reply:
When I submit it here it will also appear in Transifex. When I can do a translation directly it is better to do it right now, before we forgot to translate it.